app/_locales/en/messages.json

```diff
@@ -26,6 +36,9 @@
   "clearApprovalDataSuccess": {
     "message": "Approved website data cleared successfully."
   },
+  "airsign": {
+    "message": "AirSign"
```

Comment:
if this is the name of a company or product, it shouldnt need an i18n entry, as i expect it be rendered the same across languages

Reply:
Ok, removed them in 7b5b8ce41.